server/db/index.js

```diff
@@ -0,0 +1,85 @@
+const db = require('./database');
+const {
+    Product,
+    Category,
+    Size,
+    Gender,
+    Color,
+} = require('./product_models/index');
+
+const {
+    Order,
+    OrderStatus,
+} = require('./order_models/index');
+
+const {
+    User,
+    UserType,
+} = require('./user_models/index');
+
+////////////////////////
+// ORDER ASSOCIATIONS //
+////////////////////////
+
+// an order can only have one status
+Order.hasOne(OrderStatus);
```

Comment:
Not entirely sure if we need a hasOne relation. hasOne here would be adding the OrderID as a foreign key in the OrderStatus table. This would add additional rows for each OrderID we have in the Orders table I believe.

Reply:
https://sequelize.readthedocs.io/en/latest/docs/associations/#hasone

Reply:
Since tables like OrderStatus, Size, Color are reference tables I dont think they need a foreign key.